contracts/recoveryContracts/RecoveryWalletSimple.sol

```diff
@@ -62,6 +58,7 @@ contract RecoveryWalletSimple is IERC721Receiver, ERC1155Receiver {
     uint256 value,
     bytes calldata data
```

Comment:
Shold this one use memory?

Reply:
https://github.com/BitGo/eth-multisig-v4/blob/120/contracts/recoveryContracts/RecoveryWalletSimple.sol#L106

This one also

Reply:
I think calldata is the cheaper option and often often used for large arrays or strings. So I think here it should be calldata only